aodndata/acorn/handler.py

```diff
@@ -79,6 +79,7 @@ class AcornHandler(HandlerBase):
     def __init__(self, *args, **kwargs):
         super(AcornHandler, self).__init__(*args, **kwargs)
         self.allowed_extensions = ['.nc']
+        self.opendap_root = self.resolve_params['opendap_root']
```

Comment:
If it's in the config, it will not be in the params, so you can access it like so:

```python
self.opendap_root = self.config.pipeline_config['global'].get('opendap_root', 'http://thredds.aodn.org.au/thredds/dodsC/')
```